scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/core/service/impl/GitRepositoryService.java

```diff
@@ -83,7 +86,9 @@ public void cloneRepository(String gitUrl, String requester) throws InvalidGitUr
         repositoryEntity.setCreationDate(creationDate);
         repositoryEntity.setFolderName(folderName);
   
-        jGitService.clone(path + folderName, gitUrl);
+        String imageId = jGitService.clone(path + folderName, gitUrl);
```

Comment:
Am I correct that I should be building images as soon as the repository is Cloned?

Reply:
Yes, but why is the jgitsevice returning an image id?